test/e2e/cluster_size_autoscaling.go

```diff
@@ -29,15 +28,12 @@ import (
 )
 
 const (
-	scaleUpTimeout   = 20 * time.Minute
-	scaleDownTimeout = 30 * time.Minute
+	scaleTimeout = 2 * time.Minute
 )
 
 // [Feature:ClusterSizeAutoscaling]: Cluster size autoscaling is experimental
```

Comment:
This is not true. The feature no longer requires GCM enabled, though we still want to run this test in the dedicated suite.